docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+
+Refer to the [Using configtx.yaml to build a channel configuration](create_channel_config.html) tutorial to learn more about this file. However, the following three sections require specific configuration in order to create a channel without a system channel.
+
+**Note:** Peer organizations are not required when you initially create the channel, but if you know them, it is recommended to add them now to the Profiles `Organizations:` and `Applications:` sections to avoid having to update the channel configuration later.
+
+#### Organizations:
+
+Provide the orderer organization MSP and any peer organization MSPs that are known. Also provide the endpoint addresses of each ordering node.
+
+- **Organizations.OrdererEndpoints:**
+
+For example:
+```
+OrdererEndpoints:
+            - "Host1:7080"
+            - "Host2:7080"
+            - "Host3:7080"
+```
+
+#### Orderer:
+
+- **Orderer.OrdererType** Set this value to `etcdraft`. As mentioned before, this process does not work with Solo or Kafka ordering nodes.
+- **Orderer.EtcdRaft.Consenters** Provide the list of orderer hostnames and ports, along with their TLS certificate and private key, that are considered active members of the consenter set. All orderers that are listed in this section will become active "consenters" on the channel when they join the channel.
+
+For example:
+```
+EtcdRaft:
+    Consenters:
+        - Host: Host1
+          Port: 7090
+          ClientTLSCert: ../config/organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/signcerts/tls-cert.pem
+          ServerTLSCert: ../config/organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/signcerts/tls-cert.pem
+        - Host: Host2
+          Port: 7091
+          ClientTLSCert: ../config/organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn2.ordererOrg1.example.com/tls/signcerts/tls-cert.pem
+          ServerTLSCert: ../config/organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn2.ordererOrg1.example.com/tls/signcerts/tls-cert.pem
+        - Host: Host3
+          Port: 7092
+          ClientTLSCert: ../config/organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn3.ordererOrg1.example.com/tls/signcerts/tls-cert.pem
+          ServerTLSCert: ../config/organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn3.ordererOrg1.example.com/tls/signcerts/tls-cert.pem
+```
+
```

Comment:
Mention that for simplicity in this example, that each orderer uses the same TLS cert for both server and client, although this need not be the case.